.github/workflows/ci_cd.yml

```
@@ -615,3 +631,31 @@ jobs:
         with:
           cname: ${{ env.DOCUMENTATION_CNAME }}
           token: ${{ secrets.GITHUB_TOKEN }}
+
+  doc-index-stable:
+    name: "Deploy stable docs index"
+    runs-on: ubuntu-latest
+    needs: upload_docs_release
+    steps:
+      - name: "Install Git and clone project"
+        uses: actions/checkout@v4
+      - name: "get version of library"
+        shell: bash
+        run: |
+          echo "FULL_VERSION=$(cat VERSION)" >> $GITHUB_ENV
```

```suggestion
      - name: "Get library version"
        run: |
          echo "FULL_VERSION=$(cat VERSION)" >> $GITHUB_ENV
```
Where is that VERSION coming from @Revathyvenugopal162 ? I haven't defined it anywhere...

---

solved it.